src/index.ts

```diff
@@ -260,14 +260,14 @@ function getRouteParams(route: Route): string {
 
 function getRouteData(route: Route): string {
   if (route.method === 'post' && route.body) {
-    return 'data'; // the name `data` is hardcoded for this param in `getRouteArguments`
+    return 'JSON.stringify(data)'; // the name `data` is hardcoded for this param in `getRouteArguments`
   }
 
   let s = '{\n';
   s += route.params
     .filter(param => param.inBody)
     .map(param => {
-      return `          ${param.name}`;
+      return `          ${param.name}: JSON.stringify(${param.name})`;
```

Comment:
I don't think `JSON.stringify` achieves the purpose. Wasn't the idea to use `encode` from `io-ts` instead?

Reply:
was it? no one wrote anything in the issue :(

Reply:
Yeah, I think that's the point. Sometimes using `JSON.stringify` achieves the same goal, but more generally the problem is that custom encoders are not respected.

I'll make an example inline.